string name
---
int32 instance_id
int32 description_id
geometry_msgs/PoseWithCovarianceStamped pose_cov_stamped
Region region
